docs/spec/README.md

```diff
@@ -1,12 +1,12 @@
 # Helm Controller
 
-The Helm Controller is a Kubernetes operator, allowing one to declaratively manage Helm chart
+The Helm Controller is a Kubernetes controller, allowing one to declaratively manage Helm chart
 releases with Kubernetes manifests.
 
 ## Motivation
 
-The main goal is to provide an automated operator that can perform Helm actions (e.g.
-install, upgrade, rollback, test) and continuously reconcile the state of Helm releases.
+The main goal is to provide an automated controller that can perform Helm actions (e.g.
```

Comment:
I think the usage of controller is OK here, as "an automated controller that can perform Helm actions" is part of what makes it an operator.